hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -834,6 +832,13 @@ private ConditionQuery constructSearchQuery(ConditionQuery query,
         return query;
     }
 
+    private String propertyValueToString(Object value) {
+        // join collection data with white space
```

Comment:
the first uppercase char is expected: Join collection items with white space if the value is Collection, or else keep the origin value.